contracts/oracles/MakerDAOOracle.sol

```diff
@@ -6,39 +6,57 @@ import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 
 contract MakerDAOOracle is IOracle, Ownable {
```

Comment:
@adamdossa since we removed all references to MakerDAO in the code, do you want to rename the contract too?

Reply:
Think it is OK to leave. This contract will only work w/ MakerDAO deployed Medianizer contracts.